modules/experimental/pages/cc-blast.adoc

```diff
@@ -0,0 +1,25 @@
+= CC Blast
+
+`Detachment ability`
+
+These units have some special means to disrupt their opponents in close combat. 
+
+For example, Plague Marines of Nurgle carry the potent daemonic disease known as Nurgle's Rot, which corrupts and debilitates those who come into contact with it.
```

Comment:
I've just spotted something that makes this example slightly awkward: In the Chaos Commander detachment in the builder, there's a _Nurgle's Rot_ Daemon Price upgrade option. Given that this is a different effect to the one we're associating with Nurgle's Rot here, there is the potential for confusion.

Any thoughts on this? The first thing that occurred to me was to perhaps rename the upgrade in the builder to something like "Nurgle's Blessing". But perhaps it is better to change the example here again, to something more generalised like:

```suggestion
For example, forces of the Chaos god Nurgle often carry potent daemonic plagues which corrupt and debilitate those who come into contact with them.
```